cfme/tests/configure/test_logs.py

```diff
@@ -130,20 +125,38 @@ def test_provider_log_level(appliance, provider, log_exists):
         initialEstimate: 1/4h
         casecomponent: Configuration
         testSteps:
-            1. Change log level to debug
+            1. Change log level to warn
```

Comment:
_Optional_ these `testSteps` do not reflect the test since first you change to `info` and then change to `warn`. But since it's clear in the actual test what's going on, I leave that as _optional_

Reply:
Thanks, I'd better change it to be clear :)